### DISCUSSION TITLE
Preview images...

### DISCUSSION THREAD

**Discussion by craouette** [Poly-Haven/Public-API#7] [General]
Hi, 
I didn't found in the api the list of preview images linked to an asset... from the website, I understand that https://cdn.polyhaven.com/asset_img/primary/{id}.png can be used, but how is it possible to find the other images? 
second point, how can I know if an asset was updated?

nice api
have a nice day,
pierre

**Comment by gregzaal** [comment]
Hi Pierre,

Thumbnail URLs are not currently part of the API, sorry. We may change these at any time and don't want to promise any kind of consistent URL structure or even image set.
